### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (5 - Conflicting goals)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#484]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

<h3>Conflicting goals</h3>
WCAG 3.0 appears to have three underlying goals.

1)	To create a document which takes a holistic view of accessibility across content, agents and tools..
2)	To create a set of guidelines that can include provisions for people with disabilities that goes beyond what is allowed using the WCAG 2 rules for inclusion (e.g objective, testable provisions etc.)
3)	To create a document in the WCAG 2.0 line that will be picked up by the field and regulatory agencies and thus given the weight and power that a standard or regulation would have.  (Hence the efforts to make the initials of the group come out to be WCAG, and its number to be 3.0 even though is the actually the first in its line.)

All of these are needed and laudable (and lauded above).   But goals 2 and 3 cannot be met by the same document.  
In order to be a standard that is picked up in regulation - it has to be objective, and all of the tests and provisions need to be objective.  One simply cannot meet this goal and still have the type of document described in (2) above.  

The authors seem to be aware of the inability to address #2 and still have objective criteria and tests. Great pains have been taken in WCAG 3 to describe all sorts of testing that is non-objective (e.g. the use of qualitative rating scales, qualitative rather than quantitative or binary measures, usability testing, etc.) in order to be able to achieve goal two.  Unfortunately, this disqualifies it from goal three.  

In the end the group will need to either rename the document and have it be a really wonderful guidance document with broad scope for including guidance provisions, or return to the WCAG 2x like criteria for selecting provisions that is needed in a standard that could be adopted in regulation.  This latter choice would, of course, put you back in the same bind as the existing WCAG 2 thread.  It is aggravating, bang-head-against-the-wall frustrating, etc. but that is the situation.  (See more detail below)

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by rachaelbradley** [comment]
Draft response:  Thank you for taking the time to review and respond to our initial draft. We agree that the approach taken in the first draft does not sufficiently balance these two requirements. We received suggestions on how to better approach this balance and are exploring those. We will keep your concerns in mind while we continue to explore ways to meet both these goals and will reach out for feedback from regulators as we progress. We are closing this issue but we welcome your input on the new approach when it is ready.

**Comment by alastc** [comment]
The group approved the response above: https://www.w3.org/2022/02/01-ag-minutes.html#t05
